homeassistant/components/isy994/const.py

```diff
@@ -465,38 +456,38 @@
         27: "factory reset",
     },
     UOM_HVAC_ACTIONS: {  # Thermostat Heat/Cool State
-        0: CURRENT_HVAC_IDLE,
-        1: CURRENT_HVAC_HEAT,
-        2: CURRENT_HVAC_COOL,
-        3: CURRENT_HVAC_FAN,
-        4: CURRENT_HVAC_HEAT,  # Pending Heat
-        5: CURRENT_HVAC_COOL,  # Pending Cool
+        0: HVACAction.IDLE.value,
```

Comment:
`mypy` seems to get awfully confused if I use the enum directly in `UOM_TO_STATES`